cli/source/app.tsx

```diff
@@ -133,24 +135,46 @@ const CliUi = () => {
 
   useEffect(() => {
     const { process, promise } = runCommand(
-      'docker-compose',
-      ['watch'],
+      'docker',
+      ['compose','watch'],
       runCommandOpts
     );
 
     promise.catch((err) => {
-      handleExit();
-      dispatch(
-        appSlice.actions.addLog({
-          type: 'default',
-          line: `docker watch failed: ${err}`,
-          time: new Date()
-        })
-      );
+      if(err.errno == -2){
```

Comment:
Can we have the number `-2` as an enum in another file? Having a magic number in the code can lead to some problems down the lines and it is also not very expressive of what to expect with the error. We can have a file where we define all the constants and error codes. This way we know what error we are checking for.